.github/workflows/rust-ci-cd.yml

```
@@ -323,318 +382,272 @@ jobs:
         run: |
-          echo "Testing memory usage during dry-run execution..."
-          /usr/bin/time -v ./projektwoche-setup install -d projektwoche > memory-usage-log.txt 2>&1
-          cat memory-usage-log.txt
+          VERSION=$(grep '^version' Cargo.toml | head -n 1 | awk -F\" '{print $2}')
+          echo "VERSION=$VERSION" >> $GITHUB_ENV
+          echo "Extracted version: $VERSION"
 
-      - name: 📁 Upload performance logs as artifacts
-        uses: actions/upload-artifact@v4
+      - name: 📦 Create Release
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.7)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)